.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org/
```

Comment:
What are you attempting ?
I have a few ideas, but will let you answer before I speculate for a fix